public/components/schemas/Transaction.yml

```diff
@@ -0,0 +1,13 @@
+description: A representation of a transaction
+allOf:
+  - type: object
+    properties:
+      transactionId:
```

Comment:
Actually on the paths stating with `/transactions`, I think the id should be called `id` only. However on the paths starting with `/requests`, when it is nested under extraInformation it should be called `transactionId`. We probably need to update the API code to be aligned on that as well.